From 92a936abafbd4350def7873c638c5cb79a6116f1 Mon Sep 17 00:00:00 2001
From: Andrew Savchenko <bircoph@gmail.com>
Date: Sun, 6 Jan 2019 19:22:33 +0300
Subject: [PATCH 11/14] gssapi-configure

Disable GSSAPI checks when --without-gssapi is passed.

(cherry picked from commit de83503ce4d561294476d97b77e62001f5812a64)
(cherry picked from commit be6f289ba273a4213fcb7b433adc51b01e7b49e6)
---
 src/external/rra-c-util/m4/gssapi.m4 | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/external/rra-c-util/m4/gssapi.m4 b/src/external/rra-c-util/m4/gssapi.m4
index c8b6a9555..9d6400422 100644
--- a/src/external/rra-c-util/m4/gssapi.m4
+++ b/src/external/rra-c-util/m4/gssapi.m4
@@ -205,7 +205,8 @@ AC_DEFUN([RRA_LIB_GSSAPI],
     [AS_IF([test x"$withval" != xyes && test x"$withval" != xno],
         [rra_gssapi_libdir="$withval"])])
 
- AS_IF([test x"$rra_reduced_depends" = xtrue],
+AS_IF([test x"$with_gssapi" != xno],
+ [AS_IF([test x"$rra_reduced_depends" = xtrue],
     [_RRA_LIB_GSSAPI_PATHS
      _RRA_LIB_GSSAPI_REDUCED],
     [AS_IF([test x"$rra_gssapi_includedir" = x \
@@ -216,4 +217,4 @@ AC_DEFUN([RRA_LIB_GSSAPI],
 
  RRA_LIB_GSSAPI_SWITCH
  _RRA_LIB_GSSAPI_EXTRA
- RRA_LIB_GSSAPI_RESTORE])
+ RRA_LIB_GSSAPI_RESTORE])])
-- 
2.34.1

